[BUG] ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

<!-- 

/!\ IMPORTANT: Before opening issue

- Please update all of your packages. I won't investigate on the issue if you are on an old version.
- Please check that you have the version for all Ts.ED dependencies (expected the logger). Many issues are related to a misconfigured packages.
- Please provide a reproducible example. The stack trace isn't enough to investigate.

If theses requirements aren't checked, I WON'T INVESTIGATE (excepted if you are a PREMIUM SPONSORS!) and issue will be CLOSED!
--> 

## Information

- **Version:** 6.x
- Packages:    
    "@tsed/ajv": "^6.114.9",
    "@tsed/common": "^6.114.9",
    "@tsed/core": "^6.114.9",
    "@tsed/di": "^6.114.9",
    "@tsed/exceptions": "^6.114.9",
    "@tsed/json-mapper": "^6.114.9",
    "@tsed/logger": "^6.1.2",
    "@tsed/mongoose": "^6.114.9",
    "@tsed/openspec": "^6.114.9",
    "@tsed/platform-express": "^6.114.9",
    "@tsed/platform-middlewares": "^6.114.9",
    "@tsed/schema": "^6.114.9",

## Error

```bash
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

services/test
├─┬ @tsed/common
│ └─┬ @tsed/platform-views
│   └─┬ consolidate
│     ├── ✕ unmet peer react-dom@^16.13.1: found 18.1.0
│     └── ✕ unmet peer react@^16.13.1: found 18.1.0
└─┬ @tsed/platform-express
  └── ✕ unmet peer body-parser@1.19.0: found 1.20.0
```
Could you please update these packages? I have tested ts.ed is compatible with new version of these packages.
This error will result in an exit code of 1

## Acceptance criteria

- [ ] compatible with new version of these packages

--- a/packages/platform/platform-express/package.json
+++ b/packages/platform/platform-express/package.json
@@ -90,7 +90,7 @@
     "@tsed/platform-views": "^6.114.9",
     "@tsed/schema": "^6.114.9",
     "@types/multer": "^1.4.5",
-    "body-parser": "1.19.0"
+    "body-parser": "^1.19.0"
   },
   "peerDependenciesMeta": {
     "@tsed/common": {

